### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by amrita-shrestha**
This PR adds documentation regarding php library required to run tests and updates documentation
added docs to install ocis-php-sdk using composer

Related issue
- https://github.com/owncloud/ocis-php-sdk/issues/19 https://github.com/owncloud/ocis-php-sdk/issues/191

### CODE REVIEW

**Comment by phil-davis** [comment]
@amrita-shrestha please rebase this so it "sees" the current master license.

**Comment by phil-davis** [comment]
I rebased just now,

**Comment by individual-it** [comment]
@amrita-shrestha are you still working on this PR?

File: tests/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+##  Requirements
+To run tests, install mentioned packages: 
+- php-curl
+- php-dom
+- php-phpdbg
+- php-mbstring
+- php-ast
+
+## Run a single integration test
+`vendor/phpunit/phpunit/phpunit --configuration ./phpunit.xml --testsuite integration --filter testGetResources`
```
Comment (individual-it) [inline tests/README.md:10]: we should put that in the make command

**Review by individual-it — CHANGES_REQUESTED** [review]

File: Makefile [hunk 2]
```diff
@@ -40,6 +40,11 @@ test-php-integration: run-ocis-with-keycloak
 	composer install
 	$(call run-with-cleanup, $(PHPUNIT) --configuration ./phpunit.xml --testsuite integration, $(MAKE) docker-clean)
 
+.PHONY: test-single-php-integration
+test-single-php-integration:             ## Run single php integration test
+	composer install
+	$(PHPUNIT) --configuration ./phpunit.xml --testsuite integration --filter $(filter-out $@,$(MAKECMDGOALS))
```
Comment (individual-it) [inline Makefile:46]: can we put that into the normal `test-php-integration` goal?
check if there are any extra arguments, and if yes add `--filter` and if not omit it

**Review by individual-it — CHANGES_REQUESTED** [review]

**Pull request merged**